src/index.html

```diff
@@ -11,7 +11,124 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <h1>HTML Forms</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="rows">
+        <legend> Personal information
+        </legend>
+        <label for="surname" >
+        Surname: <input
+        type="text"
+        name="surname"
+        id="surname"
+        autocomplete="off"
+        required >
```

Comment:
fix indents in all places like this
```suggestion
        Surname:
        <input
          type="text"
          name="surname"
          id="surname"
          autocomplete="off"
          required
        >
```